src/routes/account/[uid]/+page.server.ts

```diff
@@ -67,8 +67,7 @@ export const actions = {
             ],
             subscription_data: subscriptionData,
             mode: 'subscription',
-            success_url: `${origin}/account/${uid}?subscription_checkout_status=success?session_id={CHECKOUT_SESSION_ID}`,
-            cancel_url: `${origin}/account/${uid}?subscription_checkout_status=cancel`,
```

Comment:
got rid of the `cancel_url` since we are not using it. this is the URL you want the user to get redirected to if they **start** checkout but actually **cancel** it halfway through, NOT the page used when a user actually cancels their subscription.

Reply:
Yeah I knew that was what this parameter was, did you check that it doesn't break anything if you actually try to sign up and cancel midway through? Where does it take you now?

Reply:
i lied btw you are right lol we need it to have the back button pop up on the checkout screen, so i brought the cancel url back. this was mostly my confusion on what the cancel URL actually did